src/media/js/forms_transonic.js

```diff
@@ -143,7 +143,7 @@ define('forms_transonic',
             $region_feed.find('.feed-element').each(function(i, feed_element) {
                 data[region].push([feed_element.getAttribute('data-type'),
                                    feed_element.getAttribute('data-id')]);
-            });
+            }); // I think this has a bug in it because of regions...
```

Comment:
I think there's a bug here but I'm no JS interpreter. I'm assuming that's why jshint is mad. `region` isn't scoped to this function.